### ISSUE TITLE
Happy holidays and thank you!

### ISSUE THREAD

**Issue by jaltek** [syncany/syncany#545]
I just want to say thanks to the whole syncany team and contributors.
Thank you for this great piece of software being so much better than other solutions I have tried the last years. 
Thank you for spending your valuable (spare) time for developing a better, secure and open alternative to those "top-dog-closed-source-clients" like Dropbox etc.

Sure - it is alpha and it may have some bugs but that does not change the fact that syncany is on a good way.

Keep up the good work!
Happy holidays to you and of course a happy new year.

**Comment by pimotte** [comment]
Thanks! Happy holidays to you too:)

**Comment by binwiederhier** [comment]
Aww. Thank you so much. You have no idea  how much that means to us. 

I struggle sometimes to find the time, and you certainly noticed that Pim has been pulling 99% of the weight lately. But these kind of comments really make me want to spend more time with Syncany again. That you for sticking with us.

Happy holidays to you!!

**Comment by fa72bd53** [comment]
I can only second jaltek's post and wishes! :)

Spent my free time installing syncany on my family's machines, all syncing peacefully now.

Thank you and a happy new (gregorian calender) year!

**Comment by sukosevato** [comment]
Agreed, Syncany is the only application that checks all the check marks of how an opensource syncing application should be designed. Keep up the good work and thank you for your time! Happy holidays.

**Comment by wamatt** [comment]
I've never used Syncany but I love the thoughtfulness behind it and wish it had more traction/marketing. My issue though is it seems to be kind of unloved though not totally abandoned? 

![image](https://cloud.githubusercontent.com/assets/1271434/12155241/0c9f3ac8-b479-11e5-85f8-676fbd211600.png)

Would be great to support it and recommend it to others

**Comment by pimotte** [comment]
The current activity status is: Philipp and me try to get something done every once in a while, but since both of us have other full time commitments, there's only so much we can do. The more active periods are from when both of us just had more spare time.

More traction/marketing would be good, but directed towards developers and/or dedicated testers. We're not at a stage where were ready for the main public. Development is currently our main bottleneck, so if you know interested developers, feel free to introduce them to us, and I will make myself as available as possible to help get them started.

**Comment by cr0** [comment]
I would love to be more active again. I'm still using Syncany and I really want to improve things. However there's too much to do in rl at the moment :(

**Comment by tgharib** [comment]
Have you guys considered applying for Google Summer of Code?

**Comment by pimotte** [comment]
Mentoring in the GSoC is a commitment not to be taken lightly, and I don't feel like we have the manpower to actually support that right now.

**Comment by internetimagery** [comment]
Just have to jump in here too and say thank you and happy new year!

I've always thought a program like this should exist. Very happy you guys are putting in this time to make it a reality. Time well spent!
